HACKING-cli-image.md

```diff
@@ -0,0 +1,98 @@
+Strimzi Development CLI image
+=============================
+
+Strimzi Development CLI image is a CentOS 7 based image that contains Java 8 and all the build dependencies and tools needed to build and deploy Strimzi.
+
+In order to build Strimzi Kafka Operator project you need a running Docker daemon.
+
+If you also want to try them out, deploy the Strimzi Cluster Operator, and run a test Kafka cluster on Kubernetes, you'll need access to a Kubernetes API server.
+
+There are several locally running options for Kubernetes as explained in [Strimzi Quickstarts](https://strimzi.io/quickstarts/).
+
+Look for instructions on installing Docker, Kubernetes in those documents or on web.
+
+For using Strimzi Development CLI image you only need a working Docker daemon, and `docker` client.
+
+
+Starting the session
+--------------------
+
+First, make sure that you have the latest version of the image (best to do this every time you start a new session):
+
+    docker pull quay.io/mstruk/strimzi-dev-cli
```

Comment:
why this image is in your private repo and Quay.io? Can we make it available on Docker Hub in our Strimzi organization? I think it would be great community wise.

Reply:
Right, that's one thing to address. We would probably move it under strimzi org on DockerHub.

And there's https://github.com/mstruk/strimzi-dev-cli that's the source from which the image is built (using quay.io build integration with GitHub). We would also change ownership on this one, and give it to https://github.com/strimzi .

Reply:
yes agree

Reply:
So the question is are we all OK with that.

Reply:
Yep sorry I approved but I forgot about this. Wdyt @scholzj ? Should we move the project under strimzi org and the image in the Docker Hub Strimzi before merging this one?

Reply:
I'm a bit split about this:
* If we include this into the official repository, it should be properly tested and maintained. Whcih probably means it needs ot be used somewhere. (Perhaps in the CI?)
* On the other hand, if it has not the level of being Strimzi, should it be referred here?